src/DspCInterface.h

```diff
@@ -62,6 +62,19 @@ int readDro(DspApiEnv * env, const char * dro);
 
 /** load first-stage problem */
 void loadFirstStage(
+ 		DspApiEnv *          env,   /**< pointer to API object */
+ 		const CoinBigIndex * start, /**< start index for each row */
+ 		const int *          index, /**< column indices */
+ 		const double *       value, /**< constraint elements */
+ 		const double *       clbd,  /**< column lower bounds */
+ 		const double *       cubd,  /**< column upper bounds */
+ 		const char *         ctype, /**< column types */
+ 		const double *       obj,   /**< objective coefficients */
+ 		const double *       rlbd,  /**< row lower bounds */
+ 		const double *       rubd   /**< row upper bounds */);
+
+/** load first-stage problem with quadratic objective */
+void loadQuadraticFirstStage(
```

Comment:
Can we overload function `loadFirstStage`?

Reply:
Since this is C interface, overload is not allowed. We may use C++ for DspCinterface, then we can overload functions.